.ci/oci_chectl_olm.sh

```diff
@@ -1,6 +1,5 @@
-#!/bin/bash
```

Comment:
Maybe for shell script we have to save #!/bin/bash ? I guess they will be broken without this directive...

Reply:
My mistake